fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -1773,6 +1777,54 @@ module.exports = function(config, DB) {
           }
         )
 
+        test(
+          'unblockCodes',
+          function (t) {
+            t.plan(4)
+            var uid1 = newUuid()
+            var code1 = unblockCode()
+
+            db.consumeUnblockCode(uid1, code1)
+              .then(
+                function () {
+                  t.fail('consuming unknown code should error')
+                },
+                function (err) {
+                  t.pass('consuming unknown code errors')
```

Comment:
I looked at the other tests, and it doesn't seem like the error is checked. *shrug*